packages/cloudfoundry/src/pipeline/stages/runJob/cloudFoundryRunJob.module.ts

```diff
@@ -5,7 +5,7 @@ import { CloudFoundryRunJobStageConfig } from './CloudFoundryRunJobStageConfig';
 import { RunJobExecutionDetails } from './RunJobExecutionDetails';
 
 Registry.pipeline.registerStage({
-  accountExtractor: (stage: IStage) => stage.context.credentials,
+  accountExtractor: (stage: IStage) => [stage.context.credentials],
```

Comment:
Thanks for this.  Any chance for an automated test to demonstrate the bug/fix?

Reply:
Not sure I can to be honest - I'm surprised this even compiled in the first place.